.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!-- If this PR closes a JIRA ticket, make sure the title starts with the JIRA issue number, 
+for example JP-1234: <Fix a bug> -->
+Resolves [JP-nnnn](https://jira.stsci.edu/browse/JP-nnnn)
```

Comment:
Tickets may be coming also from RCAL, so add a line
Resolves [RCAL-nnnn](https://jira.stsci.edu/browse/RCAL-nnnn)